### PR TITLE
Updated HackHands to 1.4.8

### PR DESCRIPTION
**Pull request by sergiopvilar**

### CODE REVIEW

**Comment by Ngrd** [comment]
Thank you for the contribution! However, the comment `# amazonaws.com is the official download host per the vendor homepage` shouldn't be removed.

File: Casks/hackhands.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask :v1 => 'hackhands' do
-  version '1.3.2.76'
-  sha256 'ba2244b15274158b08a4680dac512761c3c2b8ab2127a4e465ead10bb48fe1b9'
```
Comment (vitorgalvao) [inline Casks/hackhands.rb]: This empty line should be removed.

**Comment by vitorgalvao** [comment]
@sergiovilar Could you please explain why you removed the comment? Serious question. You’re not the first one to do it, and I’d honestly like to understand what is it about that comment that makes it seem like it’s OK to remove.

**Comment by sergiovilar** [comment]
@Ngrd I'm sorry, I'll put back.

**Comment by sergiovilar** [comment]
@vitorgalvao No reason, I just thought that the comment was not needed, sorry.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 73ea55249bdacdad404cf3d292cc812f5a95c45c. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by sergiovilar** [comment]
@vitorgalvao ok, thank you!

**Pull request closed without merging**